controllers/storagecluster/cephcluster.go

```diff
@@ -562,9 +565,28 @@ func newCephCluster(sc *ocsv1.StorageCluster, cephImage string, kmsConfigMap *co
 			// if resource profile change is in progress, then set this flag to false
 			ContinueUpgradeAfterChecksEvenIfNotHealthy: sc.Spec.ResourceProfile == sc.Status.LastAppliedResourceProfile,
 			CephConfig: getCephClusterCephConfig(sc),
+			Security: rookCephv1.ClusterSecuritySpec{
+				CephX: rookCephv1.ClusterCephxConfig{
+					Daemon: rookCephv1.CephxConfig{
+						KeyRotationPolicy: rookCephv1.KeyGenerationCephxKeyRotationPolicy,
+					},
+				},
+			},
 		},
 	}
 
+	KeyGenerationValue, ok := os.LookupEnv(keyRotationCounterEnvVarName)
+	if !ok {
+		panic("KEY_ROTATION_COUNTER env variable is not set")
+	}
+
+	KeyGeneration, err := strconv.Atoi(KeyGenerationValue)
+	if err != nil {
+		panic("can not convert KEY_ROTATION_COUNTER value to a integer")
+	}
```

Comment:
We could have a util for this as this is going to be used in multiple places

Reply:
Done